tests/plugins/toolbar/manual/aria-haspopup.md

```diff
@@ -0,0 +1,19 @@
+@bender-tags: 4.8.0, trac16893, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, language, stylescombo, format, colorbutton, font
+
+# Test scenario:
+
+Open web inspector, for each button in menu find it's 'a' element and search for `aria-haspopup` attribute.
```

Comment:
This test is totally unfriendly. It'd rather create list, in which all `[aria-haspopup]` attributes for the buttons will be listed.